source/core/VirtualInstrument.cpp

```diff
@@ -34,7 +34,7 @@ NIError VirtualInstrument::Init(TypeManagerRef tm, Int32 clumpCount, TypeRef par
     _locals->SetElementType(localsType, false);
     _clumps->Resize1D(clumpCount);
     _lineNumberBase = lineNumberBase;
-    _clumpSource = clumpSource;
```

Comment:
This change was necessary because of adding explicit to single arg constructors.
Before that, it appears the compiler was automatically handling it.   There were only 3 instances where this showed up.